src/components/MonitorList.vue

```diff
@@ -306,6 +311,49 @@ export default {
 
             this.cancelSelectMode();
         },
+
+        /**
+         * Show the confirmation dialog for deleting the selected monitors.
+         * This method triggers the display of a confirmation modal where the user can
+         * confirm or cancel the deletion of the selected monitors.
+         * @returns {void}
+         */
+        deleteSelectedDialog() {
+            // Show the confirmation dialog for deletion
+            this.$refs.confirmDeleteSelected.show();
+        },
+
+        /**
+         * Delete each selected monitor and update the UI once the deletion is complete.
+         * This method initiates the deletion process for all selected monitors, updates
+         * the user interface, and reloads the page to reflect the changes.
+         * @returns {void}
+         */
+        deleteSelected() {
+            // Delete each selected monitor
+            Object.keys(this.selectedMonitors).forEach(id => {
+                this.$root.deleteMonitor(id, (res) => {
+                    // Display a response message for the operation
+                    this.$root.toastRes(res);
+
+                    // Remove the monitor from the selection if deletion is successful
+                    if (res.ok) {
+                        delete this.selectedMonitors[id];
+                    }
+                });
+            });
+
+            // Delay for UI updates before reloading the page
+            setTimeout(() => {
+                if (!Object.keys(this.selectedMonitors).length) {
+                    window.location.reload(); // Refresh the page to show updates
+                }
+            }, 5000);
```

Comment:
Let's try to remote debug this a bit (I am currently at CCC => don't have my primary laptop with me).

Is there a response to this deletion above? UI updates should be send via socket communication..

Reply:
I copied the `deleteDialog` function in `Details.vue` and made a small adjustment. Everything works as expected, but the `/dashboard` is not updated behind the scenes, so part of the deleted data (i.e., the browser cache) remains visible on the `/dashboard` (see video).

For now, I have set a 5-second delay to refresh the web page so that the correct data is displayed, until I solve the issue.

I wil look in to `UI updates should be send via socket communication`.